scripts/prepare-upgrade-bundles.bash

```
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+GENESIS_VERSION="${GENESIS_VERSION:-v1.3.0}"
+RPC_URL="${RPC_URL:-https://rpc.testnet.fluent.xyz}"
+
+CHAIN_ID=$(curl https://rpc.testnet.fluent.xyz -H 'content-type: application/json' --data-raw '{"id":1,"jsonrpc":"2.0","method":"eth_chainId","params":[]}' | jq -r '.result')
+echo "Detected Chain ID: $CHAIN_ID"
+
+mkdir -p ./target/bundles/${GENESIS_VERSION}-$CHAIN_ID
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)